pkg/kcp/provider/aws/redisinstance/util.go

```diff
@@ -1,11 +1,98 @@
 package redisinstance
 
-import "fmt"
+import (
+	"fmt"
+	"strings"
+
+	elasticacheTypes "github.com/aws/aws-sdk-go-v2/service/elasticache/types"
+	"k8s.io/utils/ptr"
+)
 
 func GetAwsElastiCacheSubnetGroupName(name string) string {
 	return fmt.Sprintf("cm-%s", name)
 }
 
+func GetAwsElastiCacheParameterGroupName(name string) string {
+	return fmt.Sprintf("cm-%s", name)
+}
+
+func GetAwsElastiCacheParameterGroupFamily(engineVersion string) string {
+	if strings.Contains(engineVersion, "2.6") {
```

Comment:
In the case of engine 3.2.6 (https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/ParameterGroups.Redis.html#ParameterGroups.Redis.3-2-6) we might get the wrong param group family.

Reply:
change to check prefix